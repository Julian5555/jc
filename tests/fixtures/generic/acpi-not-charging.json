[{"type":"Battery","id":0,"state":"Not charging","charge_percent":100}]
