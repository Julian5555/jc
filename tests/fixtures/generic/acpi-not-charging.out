Battery 0: Not charging, 100%
